component/authenticator/pom.xml

```diff
@@ -0,0 +1,199 @@
+                <version>2.0</version>
+                <configuration>
+                    <source>1.8</source>
+                    <target>1.8</target>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.apache.felix</groupId>
+                <artifactId>maven-bundle-plugin</artifactId>
+                <extensions>true</extensions>
+                <configuration>
+                    <instructions>
+                        <Bundle-SymbolicName>${project.artifactId}</Bundle-SymbolicName>
+                        <Bundle-Name>${project.artifactId}</Bundle-Name>
+                        <Private-Package>org.wso2.carbon.identity.authenticator.voiceotp.internal</Private-Package>
+                        <Import-Package>org.apache.axis2.*,
+                            org.apache.axiom.*,
+                            org.wso2.carbon.ui.*,
+                            org.apache.commons.logging.*; version="1.0.4",
+                            org.osgi.framework,
+                            org.osgi.service.component; version="${osgi.service.import.version.range}",
+                            org.wso2.carbon.identity.application.authentication.framework.*,
+                            javax.servlet,
+                            javax.servlet.http,
+                            org.wso2.carbon.extension.identity.helper.*;
+                            version="${identity.extension.utils.import.version.range}",
+                            org.wso2.carbon.identity.event; version="${carbon.identity.version.range}",
+                            org.wso2.carbon.identity.event.event; version="${carbon.identity.version.range}",
+                            org.wso2.carbon.identity.event.services; version="${carbon.identity.version.range}",
+                            org.wso2.carbon.identity.governance.service.notification;
+                            version="${identity.governance.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.handler.event.account.lock.service;
+                            version="${carbon.identity.account.lock.handler.imp.pkg.version.range}",
+                            org.wso2.carbon.identity.handler.event.account.lock.exception;
+                            version="${carbon.identity.account.lock.handler.imp.pkg.version.range}",
+                            *;resolution:=optional
+                        </Import-Package>
+                        <Export-Package>
+                            !org.wso2.carbon.identity.authenticator.voiceotp.internal,
+                            org.wso2.carbon.identity.authenticator.voiceotp.*;
+                            version="${carbon.identity.package.export.project.version}"
+                        </Export-Package>
+                    </instructions>
+                </configuration>
+            </plugin>
+            <plugin>
+                <groupId>org.jacoco</groupId>
```

Comment:
You can define this dependency in parent pom as bellow.
```
<dependency>
  <groupId>org.jacoco</groupId>
  <artifactId>org.jacoco.agent</artifactId>
  <classifier>runtime</classifier>
  <scope>test</scope>
  <version>${jacoco.version}</version>
</dependency>
```

Reference 
Parent pom : https://github.com/wso2-extensions/identity-outbound-auth-uaepass/blob/main/pom.xml
Child pom : https://github.com/wso2-extensions/identity-outbound-auth-uaepass/blob/main/components/uaepass-authenticator/pom.xml

Reply:
Having it same as the sms otp authenticator